packages/react-scripts/utils/isReactScriptsLinked.js

```diff
@@ -0,0 +1,7 @@
+var path = require('path');
+var fs = require('fs');
+
+module.exports = () => {
+  var localReactScriptsPath = path.resolve('node_modules/react-scripts');
+  return fs.existsSync(localReactScriptsPath) && fs.lstatSync(localReactScriptsPath).isSymbolicLink();
```

Comment:
My apologies, I missed that note. I made an assumption.

Is the call necessary though? We still may be able to do without it.